src/seir.py

```diff
@@ -0,0 +1,73 @@
+import opencor as oc
+
+
+def main():
+    # Open and run a 300-day simulation using the SEIR model.
+
+    simulation = oc.open_simulation(
+        'https://models.physiomeproject.org/workspace/5d4/rawfile/b973009caf430b1636cfd1133a23bfd570a85691/COVIDSim.cellml')
```

Comment:
should be loading a SED-ML with the appropriate simulation algorithm defined, etc.

Reply:
Doh, good point! Going to do that.

Reply:
Done in PR #2.